.github/workflows/code-hygiene.yml

```
@@ -0,0 +1,73 @@
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install dependencies
+        run: npm ci
+      - name: Build
+        run: npm run build
+      - name: Start preview server
+        run: npm run preview &
+      - name: Wait for server
+        run: npx wait-on http://localhost:4173
+      - name: Run Lighthouse CI
+        run: npx lhci autorun --collect.url=http://localhost:4173 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)